testcrate/benches/mem.rs

```diff
@@ -0,0 +1,154 @@
+#![feature(test)]
+
+extern crate test;
+use test::{black_box, Bencher};
+
+extern crate compiler_builtins;
+use compiler_builtins::mem::{memcmp, memcpy, memmove, memset};
+
+fn memcpy_builtin(b: &mut Bencher, n: usize) {
+    let v1 = vec![1u8; n];
+    let mut v2 = vec![0u8; n];
+    b.iter(|| {
```

Comment:
minor nit: set `b.bytes = n as u64;`?

Reply:
Nice, that's much better than doing the math normally.